package.json

```diff
@@ -14,6 +14,9 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "sls-offline": "npm run build && serverless offline start",
+    "deploy:dev": "npm run build && serverless create_domain --stage dev && serverless deploy --stage dev",
```

Comment:
@shogo-mitomo do we need to add additional config to `serverless.yml` to support this `create_domain` command? :bowing_man:

Reply:
@yashmurty Oh, you are right 💦 I added them. Please check 🙇

Reply:
@shogo-mitomo Thank you! :bowing_man: 

BTW, I added you as `collaborator` to this repository, since it's already `approved` PR, can you go ahead and merge it? 
Let me know if there is any issue. :pray: